packages/jellyfish-api-core/__tests__/category/account/getTokenBalances.test.ts

```diff
@@ -117,7 +120,7 @@ describe('Account', () => {
     })
     await container.generate(2)
 
-    const dvmAddr = await container.call('getnewaddress', ['', 'bech32'])
+    const dvmAddr = await container.getNewAddress('dvm', 'legacy')
```

Comment:
hmm.. don't get idea why change bech32 -> legacy here? 🤔

Reply:
No specific reason, we have updated this line before in my transfer domain fix (only allow legacy and bech32 address ). and this line comes with conflict while rebasing, so I kept my version for consistency (As i have used same logic in transferDomain.test.ts as well)